osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs

```diff
@@ -123,12 +123,12 @@ protected override void LoadComplete()
                     indexInCurrentComboBindable.BindValueChanged(_ => updateComboIndex(), true);
 
                     comboIndexBindable = comboInfo.ComboIndexBindable.GetBoundCopy();
-                    comboIndexBindable.BindValueChanged(_ => updateColour(), true);
-
                     comboIndexWithOffsetsBindable = comboInfo.ComboIndexWithOffsetsBindable.GetBoundCopy();
-                    comboIndexWithOffsetsBindable.BindValueChanged(_ => updateColour(), true);
 
+                    comboIndexBindable.ValueChanged += _ => updateColour();
```

Comment:
Why not use `BindValueChanged` as it was before? This seems different to how we do event binding everywhere else.

Reply:
Would have to move things to use `BindValueChanged` back (with last one running immediately):
```diff
diff --git a/osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs b/osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs
index 6b75696d23..b021258c97 100644
--- a/osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs
+++ b/osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs
@@ -125,10 +125,9 @@ protected override void LoadComplete()
                     comboIndexBindable = comboInfo.ComboIndexBindable.GetBoundCopy();
                     comboIndexWithOffsetsBindable = comboInfo.ComboIndexWithOffsetsBindable.GetBoundCopy();

-                    comboIndexBindable.ValueChanged += _ => updateColour();
-                    comboIndexWithOffsetsBindable.ValueChanged += _ => updateColour();
                     skin.SourceChanged += updateColour;
-                    updateColour();
+                    comboIndexBindable.BindValueChanged(_ => updateColour());
+                    comboIndexWithOffsetsBindable.BindValueChanged(_ => updateColour(), true);
                     break;
             }
         }
```

Either way works really.

Reply:
I don't understand what you're gaining from moving the `BindValueChanged` calls bellow the `skin.SourceChanged`. All you had to do was drop the `true` argument from the first `BindValueChanged`, like we've done everywhere else in the codebase.